features/division.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Division
+  As a mathematician
+  I want my calculator to divide numbers
+  So I can know their quotient
+
+  Scenario: Dividing numbers to get the quotient
+    When I divide two numbers
+	Then I get the quotient

```

Comment:
I don't know what's going on with the spacing here. It happened in every feature file, so maybe an editor issue? No points were taken off. Just be on the lookout for it.